python/ray/ml/predictors/integrations/torch/torch_predictor.py

```diff
@@ -112,10 +116,10 @@ def predict(
                 format of ``feature_columns``, or be a single dtype, in which
                 case it will be applied to all tensors.
                 If None, then automatically infer the dtype.
-            unsqueeze_feature_tensors (bool): If set to True, the features tensors
-                will be unsqueezed (reshaped to (N, 1)) before being concatenated into
-                the final features tensor. Otherwise, they will be left as is, that is
-                (N, ). Defaults to True.
+            unsqueeze (bool): If set to True, the features tensors will be unsqueezed
+                (reshaped to (N, 1)) before being concatenated into the final features
+                tensor. Otherwise, they will be left as is, that is (N, ).
+                Defaults to True.
```

Comment:
Nit: out of scope for this PR, but this explanation doesn't really make sense for multi-dimensional inputs

Reply:
We could change the description, yeah. We are now calling `unsqueeze` which should work with an arbitrary number of dimensions and not just 2, like the `view` call we were using before.

Reply:
Agreed the semantics here need to be changed. Though this is out of scope for this PR- this PR is just updating the docstring to match the same as the argument name.